packages/preact/test/index.test.tsx

```diff
@@ -17,6 +17,16 @@ describe("@preact/signals", () => {
 		render(null, scratch);
 	});
 
+	describe("inheritance", () => {
```

Comment:
As we have `preact-render-to-string` in our deps, should we add a test for `<p>{signal}</p>`?

Reply:
Would the test that you're suggesting be different than the SSR tests in [`packages/preact/test/ssr.test.tsx`](https://github.com/preactjs/signals/blob/55d5becbb41c2faec62b9588937f8c74f0d65152/packages/preact/test/ssr.test.tsx#L9)?

Reply:
Aha, updating initially broke the tests?

Reply:
Yeah, I can't remember which ones specifically, but iirc, 7 of the text binding tests were failing.

The inheritance tests also might be redundant, since duplicate tests are ran in `packages/core/test/signal.test.tsx`. My thought process for adding those tests was that modifying the Signal prototype may potentially mess with `instanceof`.

Though thinking about it now, it might be better to just remove those tests.